team.html

```diff
@@ -103,6 +124,7 @@ <h2 id="overview">Overview</h2>
             <li><a href="#coordination-committee">Coordination committee</a></li>
             <li><a href="#project-manager">Project managers</a></li>
             <li><a href="#lead-developer">Lead developers</a></li>
+            <li><a href="#module-maintainer">Module maintainers</a></li>
```

Comment:
Would call it "Sub-package maintainer", which I think is consistent with Astropy's definition...